### DISCUSSION TITLE
[Nemo] Add "Alt+D" as a shortcut to modify paths

### DISCUSSION THREAD

**Discussion by osdcdkje** [linuxmint/discussions#433] [Ideas]
Web browsers allow users to use the keyboard shortcuts "Alt+D" or "Ctrl+L" to focus the address bar. In Nemo, I can just use "Ctrl+L" to change the path. "Alt+D" is a better keyboard shortcut than "Ctrl+L", the keys are closer together and can be easily pressed with one hand without much attention.

**Comment by mtwebster** [comment]
For what it's worth, you can also use `/` and `~` to trigger path entry
